### ISSUE TITLE
Write Integration Tests for Utils Modules

### ISSUE THREAD

**Issue by AceTheCreator** [opensource9ja/dnotebook-react#15]
The utils modules have not been tested yet, it would be nice if an integration test is implemented

**Comment by AceTheCreator** [comment]
@steveoni @risenW Since you guys are re-writing danfo.js with typescript, should  I stop working on Dnotebook for now and look into other projects or continue working on Dnotebook?

**Comment by risenW** [comment]
> @steveoni @risenW Since you guys are re-writing danfo.js with typescript, should I stop working on Dnotebook for now and look into other projects or continue working on Dnotebook?

Yes, let's pause this for now. I'll ping you as soon as we are ready to continue. We have some new features we intend to add to Dnotebook too.
